development/mimir-with-otlp/config/otel-collector.yaml

```diff
@@ -0,0 +1,32 @@
+exporters:
+  prometheus:
+    endpoint: :8083
+  otlphttp:
```

Comment:
that would be a bit easier to read as `otlp_http`

Reply:
I agree with you! 

But this is the config for `otel-collector` and unfortunately they named it `otlphttp`: https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/otlphttpexporter